ow_lander/launch/planning.launch

```diff
@@ -49,6 +49,9 @@
     <!-- == launch the trajectory publishing node ========= -->
     <node pkg="ow_lander" name="trajectory_publisher" type="trajectory_publisher.py" output="screen">
     </node>
+    <!-- == launch the mechanical power publishing node ========= -->
+    <node pkg="ow_lander" name="mechanical_power_arm" type="mechanical_power_arm.py" output="screen">
```

Comment:
This is unrelated to planning. Shouldn't it be in common.launch instead of planning.launch?

Reply:
Done!